docs/provisioning_guide/src/asciidoc/_chapters/script_install.adoc

```diff
@@ -196,7 +194,7 @@ You can monitor the SQ shell log file : /home/trafodion/apache-trafodion-1.3.0-i
 Startup time  0 hour(s) 1 minute(s) 9 second(s)
 Apache Trafodion Conversational Interface 1.3.0
 Copyright (c) 2015 Apache Software Foundation
->> initialize trafodion;
+>> 
```

Comment:
Looks like we have a mystery command now that sqlci completed. 

I'm guessing you wanted to delete this prompt and also the "SQL opteration complete" message after it?